apps/frontend-manage/src/pages/courses/[id].tsx

```diff
@@ -286,16 +286,6 @@ function CourseOverviewPage() {
                 points: course.averageActiveScore?.toFixed(2),
               })}
             </div>
-            <div className="mt-1">
```

Comment:
as we dont want to collect the data for participants that do not participate on the leaderboard, they do not get an entry on the LB, and these numbers would be the same for both cases (total participants and active participants)

Reply:
makes sense - should we maybe also remove the internationalization strings from the corresponding files at the same time in order to avoid having leftover data in there?